api/README.md

```diff
@@ -109,3 +109,13 @@ If everything was setup correctly, you should see a response that resembles the
 ### Misc. Notes
 
 - [Swagger](https://swagger.io/) XML Comments have been enabled to provide better API Documentation.  This means that warnings will be generated for public undocumented public types and members.  By default, this project disables warnings.  Documentation to enable warnings can be found [here](https://docs.microsoft.com/en-us/aspnet/core/tutorials/getting-started-with-swashbuckle?view=aspnetcore-3.1&tabs=visual-studio-code)
+
+# Docker local development
```

Comment:
Can you add some comment to present this as an option and maybe list what they need to have setup locally?